### DISCUSSION TITLE
Citing ONNX?

### DISCUSSION THREAD

**Discussion by alexbw** [onnx/onnx#1441] [Q&A]
What's the appropriate way to cite ONNX in an academic publication?

**Comment by houseroad** [comment]
We don't have a paper to cite yet. I would suggest citing the homepage: https://github.com/onnx/onnx

**Comment by ieee8023** [comment]
Can you endorse a common title and author list for the citation? How about this:

ONNX. Open Neural Network Exchange. 2017, https://github.com/onnx/onnx.

**Comment by breandan** [comment]
How about something like this?

```
@misc{bai2019,
    author = {Bai, Junjie and Lu, Fang and Zhang, Ke and others},
    title = {ONNX: Open Neural Network Exchange},
    year = {2019},
    publisher = {GitHub},
    journal = {GitHub repository},
    howpublished = {\url{https://github.com/onnx/onnx}},
    commit = {94d238d96e3fb3a7ba34f03c284b9ad3516163be}
}
```

@bddppq @houseroad @linkerzhang

**Comment by daniel-gonzalez-cedre** [comment]
I'm also interested in how to cite ONNX in a publication. Right now, I'm deferring to @breandan's bibtex above.

**Comment by askhade** [comment]
@ebarsoum @prasanthpul any thoughts?

**Comment by prasanthpul** [comment]
@ibrahimhaddad for LF AI recommendation

**Comment by yarikoptic** [comment]
I think the best proceed ATM would be to "link" this repository to zenodo.org and make it produce citeable references for every release you publish. Then the "ultimate publication" could just reference zenodo reference and improve your h-factor ;)

Some tools which could help along the way for that: https://github.com/all-contributors/ which provides a bot to add contributors from comments in PRs etc.
Also a brand new fresh one (disclaimer: I am related ;)): https://github.com/con/tributors/ which aims to simplify management across all-contributors/.zenodo.json etc.

Since it is Python, I would also advise to look into "integrating" with duecredit: http://duecredit.org/ (again the same disclaimer as to con/tributors) ;)

**Comment by ChristopherBrix** [comment]
Has this been resolved? I couldn't find a published paper or preferred citation format. Maybe you could include one at the end of the Readme?

**Comment by andife** [comment]
I think that topic should be discussed: Maybe we should use https://citation-file-format.github.io/

**Comment by lauramartinho** [comment]
They added a citation link in their official website

https://onnxruntime.ai/docs/reference/citing.html

```
@misc{onnxruntime,
  title={ONNX Runtime},
  author={ONNX Runtime developers},
  year={2021},
  howpublished={\url{https://onnxruntime.ai/}},
  note={Version: x.y.z}
}

```

**Reply by justinchuby** [reply]
Note that ONNX and ONNX Runtime are two related but different projects
